src/components/Section.js

```diff
@@ -56,7 +56,8 @@ class Section extends Component {
         paddingTop: mjAttribute('padding-top'),
         paddingBottom: mjAttribute('padding-bottom'),
         paddingRight: mjAttribute('padding-right'),
-        paddingLeft: mjAttribute('padding-left')
+        paddingLeft: mjAttribute('padding-left'),
+        padding: mjAttribute('padding')
```

Comment:
@iRyusa Should this also be the same on `MJMLColumnElement`? As currently it's at the bottom of the stack too.